css/style.css

```diff
@@ -470,3 +470,672 @@
+  flex-direction: row;
+  flex-grow: 0;
+  width: 210px;
+  height: 71px;
+}
+
+.bag-page {
+  display: flex;
+  flex-direction: row;
+  flex-grow: 0;
+  width: 62px;
+  height: 65px;
+  padding-bottom: 10px;
+}
+
+.bag-page__img {
+  display: flex;
+  width: 36px;
+  height: 53px;
+  background: url(https://i.ibb.co/zQ7Kmns/bag.png) no-repeat;
+  background-size: contain;
+  padding: 0;
+  margin: 10px 0 0 1px;
+}
+
+.bag-text {
+  display: flex;
+  flex-flow: row wrap;
+  flex-grow: 0;
+  width: 155px;
+  height: 65px;
+}
+
+.bag-text__numeral {
+  display: flex;
+  flex-grow: 0;
+  width: 69px;
+  height: 37px;
+  margin: 0;
+}
+
+.bag-text__text {
+  display: flex;
+  flex-grow: 0;
+  width: 63px;
+  height: 22px;
+  margin: 15px 0 0 12px;
+}
+
+.bag-text__subtext {
+  display: flex;
+  width: 115px;
+  height: 28px;
+  margin: 2px 0 0 2px;
+}
+
+/* ---ссылка тест и кнопка заказать--- */
+
+.sidebar__info {
+  display: flex;
+  flex-flow: row nowrap;
+  align-items: center;
+  width: 352px;
+  height: 40px;
+  margin: 0 auto;
+  text-transform: uppercase;
+}
+
+.info__image {
+  display: flex;
+  width: 22px;
+  height: 22px;
+  background: url(https://i.ibb.co/zFFHDgv/Group-15.png) no-repeat;
+  background-size: 100%;
+  margin: 0 0 0 6px;
+}
+
+.info__link {
+  display: flex;
+  align-items: center;
+  justify-content: flex-start;
+  width: 137px;
+  height: 24px;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 13px;
+  line-height: 16px;
+  color: #fff;
+  margin: 0 24px 0 9px;
+}
+
+.info__link:hover {
+  font-size: 14px;
+  font-weight: bold;
+  text-align: center;
+  text-decoration: none;
+  color: #000;
+  cursor: pointer;
+  padding: 0;
+  border: 0;
+}
+
+.info__button {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  font-size: 13px;
+  text-align: center;
+  font-weight: bold;
+  border: 2px solid #fff;
+  color: #fff;
+  background-color: #f00;
+}
+
+.info__button:hover {
+  font-size: 14px;
+  color: #fff;
+  border: 3px solid #fff;
+  background-color: #f00;
+  cursor: pointer;
+  font-weight: bold;
+  padding: 0;
+}
+
+.info__button:focus {
+  color: #fff;
+  border: 1px solid #fff;
+  background-color: #f00;
+  cursor: pointer;
+  font-weight: bold;
+  padding: 0;
+  outline: none;
+  box-shadow: 0 0 0 3px lightskyblue;
+}
+
+.info__button:active {
+  background-color: #fff;
+  border-color: #000;
+  color: #000;
+  font-size: 12px;
+}
+
+/* --------------------------------------------Секция третья-------------------------------------------- */
+
+.sections__frame {
+  display: flex;
+  flex-flow: row wrap;
+}
+
+.frame {
+  display: flex;
+  flex-flow: row wrap;
+  width: 1440px;
+  height: 900px;
+  margin: 0 auto;
+  border: 1px solid blue;
+}
+
+.frame__sidebar {
+  display: flex;
+  flex-flow: column wrap;
+  width: 440px;
+  height: 900px;
+  font-family: Montserrat;
+  font-style: normal;
+  font-weight: 600;
+  color: #fff;
+  background: #f00;
+  background-size: 100%;
+   
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Hi-Tesla/pullRequest?prid=5726736)